### PR TITLE
Add sendWithReplyBlocking overload taking a Duration.

### PR DESCRIPTION
**Pull request by kubo39**

### CODE REVIEW

File: source/ddbus/package.d [hunk 1]
```diff
@@ -1,3 +1,5 @@
+module ddbus;
```
Comment (WebFreak001) [inline source/ddbus/package.d]: I think this makes the code more clear and should definitely be done, but not in the scope of this PR

**Review by WebFreak001 — CHANGES_REQUESTED** [review]
This is a good PR, but could you please make the changes to the examples folder in a separate PR? Also I'm not sure if the added module is scope of this PR either. Well it is actually better to have it there because I sometimes had dmd complain about files like that too but it should be done separately from this.

You already made a lot of commits, maybe you should squash your code or make a new branch based off this master branch.

So actually what I think is more appropriate for this PR is all the changes in ddbus.thin and changing one line in the existing example.d changing the 3000 (well and importing the duration call)

**Comment by kubo39** [comment]
Thank you for your review, fixed and create another PR #31.

**Review by WebFreak001 — APPROVED** [review]

**Comment by WebFreak001** [comment]
trivial change, merging without further feedback

**Pull request merged**